tests/drivers/dma/chan_blen_transfer/src/test_dma.c

```diff
@@ -93,7 +108,7 @@ void test_dma_m2m_chan0_burst8(void)
 
 void test_dma_m2m_chan1_burst8(void)
 {
-	zassert_true((test_task(1, 8) == TC_PASS), NULL);
+	zassert_true((test_task(0, 8) == TC_PASS), NULL);
```

Comment:
I don't this this is a clean change.
IIUC, this makes `test_dma_m2m_chan0_burst8` testing the same as `test_dma_m2m_chan1_burst8`.
So chan0 is not tested anymore fro those who need it

Reply:
It could be a test_task(FIRST_DMA_CHANNEL, 8) and another one test_task(LAST_DMA_CHANNEL, 8) (idem for burst16)

Reply:
sorry I forget to revert my debug change, change it back